prow/pilot-presubmit.sh

```diff
@@ -50,6 +50,9 @@ bazel build //...
 echo '=== Go Build ==='
 ./bin/init.sh
 
+echo '=== Code Format Check ==='
+./bin/fmt.sh
```

Comment:
I left comments in the original PR that created this. but how is this going to do anything/'fail' if needed ?